### PR TITLE
Remove NamedPreparedStatement usage to maintain backward compatibility

### PR DESCRIPTION
**Pull request by Buddhimah**

### CODE REVIEW

File: components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/dao/RoleDAO.java [hunk 1]
```diff
@@ -87,16 +84,12 @@ public static RoleDAO getInstance() {
         List<RoleInfo> externalRoles = new ArrayList<>();
         Map<String, Integer> repetition = new HashMap<>();
         repetition.put(HYBRID_DOMAINS, 4);
```
Comment (ashensw) [inline components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/dao/RoleDAO.java:86]: We don't need this anymore?

File: components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/dao/RoleDAO.java [hunk 2]
```diff
@@ -37,33 +36,31 @@
...
+                    + "? AND UM_TENANT_ID = ?)";
 
     public static final String DELETE_TENANT_ADMIN_ROLE_DATA =
-            "DELETE FROM UM_ROLE_PERMISSION WHERE UM_ROLE_NAME=:UM_ROLE_NAME; AND UM_TENANT_ID=:UM_TENANT_ID; AND "
+            "DELETE FROM UM_ROLE_PERMISSION WHERE UM_ROLE_NAME = ? AND UM_TENANT_ID = ? AND "
                     + "UM_DOMAIN_ID IN (SELECT UM_DOMAIN_ID FROM UM_DOMAIN WHERE UM_DOMAIN_NAME = "
-                    + ":UM_DOMAIN_NAME; AND UM_TENANT_ID=:UM_TENANT_ID;)";
+                    + "? AND UM_TENANT_ID = ?)";
 
     public static final String HYBRID_DOMAINS = "HYBRID_DOMAINS";
```
Comment (ashensw) [inline components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/dao/RoleDAO.java:58]: We don't need this anymore?

File: components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/dao/RoleDAO.java (hunk 1 above)
Comment (Buddhimah) [reply components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/dao/RoleDAO.java]: Removed

File: components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/dao/RoleDAO.java (hunk 2 above)
Comment (Buddhimah) [reply components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/dao/RoleDAO.java]: Removed

**Review by ashensw — APPROVED** [review]

**Pull request merged**